task-sdk/src/airflow/sdk/_vendor/.gitignore

```diff
@@ -0,0 +1 @@
+*/*.py
```

Comment:
```suggestion
*
```

Should not we ignore **everything** here ? Some of the vendored code might have resources.

Reply:
We should also exclude (!)the __init__.py

Reply:
And speaking of "_vendor" name - I think we should not use`_vendor` here as a name - even if we are using `vendoring` tool. Can we configure (mock?) the vendoring tool to use something different (_shared) sounds like a way better name). The `vendor` meaning is very tightly connected with `3rd party` but here we are `vendoring-in` our own code so we are really `1st party` here.

Reply:
Yeah it's already a config setting in pyproject.toml

```toml
[tool.vendoring]
destination = "src/airflow/_vendor/"
requirements = "src/airflow/_vendor/vendor.txt"
namespace = "airflow._vendor"
```

Reply:
We do have a quirk we have to avoid here -- if we have `shared/logging/src/logging/__init__.py` this would end up conflicting with the stdlib in dev using this approach. Perhaps another reason to suggest relative imports.

I guess the main issue with relative imports is are we okay with no imports between shared libs? For example, it might be nice to be able to use the timezone/date parsing stuff in the logging one, but I don't think we can do that when symlinking?